modules/ocf/templates/munin/ocf-plugin-conf.erb

```diff
@@ -0,0 +1,3 @@
+[memory]
+env.committed_warning :<%= (0.5 * 1e6 * @memorysize_mb.to_f).to_i %>
+env.committed_critical :<%= (0.8 * 1e6 * @memorysize_mb.to_f).to_i %>
```

Comment:
warning at 50% memory seems bad, that means we're planning to waste half of our memory

maybe 80% and 90% is more reasonable?